spec/intro.dd

```diff
@@ -2,11 +2,13 @@ Ddoc
 
 $(SPEC_S Introduction,
 
-        $(P The D programming language is a general purpose systems programming language.
-        To that end, a D program is a collection of modules that can be compiled separately
-        to native code that is combined with libraries and compiled C code by a linker to create a
-        native executable.
-        )
+$(P D is a general-purpose systems programming language with a C-like syntax that compiles to native code. 
+It is strongly typed and supports both automatic (garbage collected) and manual memory management. 
+D programs are structured as modules that can be compiled separately and linked with external libraries 
+to create native libraries or executables.)
+
+$(P This document is the reference manual for the D Programming Language. For more information and 
+other documents, see dlang.org.)
```

Comment:
This text is also used to render the PDF and mobile book, so all links must use Ddoc macros. Though in this case I would simply skip the last sentence as it will be weird on paper?

Reply:
I wasn't sure of the syntax for a link. I have updated it - is that the right syntax? I will have a look at the PDF output to see what this comes out as, but I would expect that PDFs should also show the link.